compiler/AST/stmt.cpp

```diff
@@ -51,7 +54,27 @@ void codegenStmt(Expr* stmt) {
     if (printCppLineno && stmt->linenum() > 0)
         info->cStatements.push_back(zlineToString(stmt));
     if (fGenIDS)
-      info->cStatements.push_back(idCommentTemp(stmt));
+      info->cStatements.push_back("/* " + numToString(stmt->id) + " */ ");
```

Comment:
This line change for the C backend just looks like a merge error - I think it should just be 

   info->cStatements.push_back(idCommentTemp(stmt));